covid19-etl/etl/idph_vaccine.py

```diff
@@ -0,0 +1,268 @@
+        }
+        age_group_mapping = {"65+": "greater than 65"}
+        if prop_name == "Race" and value in race_mapping:
+            return race_mapping.get(value)
+        if prop_name == "Gender" and value in gender_mapping:
+            return gender_mapping.get(value)
+        if prop_name == "AgeGroup" and value in age_group_mapping:
+            return age_group_mapping.get(value)
+        return value
+
+    def parse_group_clinical_demographic(self, props_mapping, props_value):
+        key_props_name = ["AgeGroup", "Race", "Gender"]
+        key_props = {}
+        for k in key_props_name:
+            key = props_mapping.get(k)
+            if key is not None:
+                key_props[key] = props_value.get(k)
+        props_data = {}
+        for (k, v) in props_mapping.items():
+            if k in props_value:
+                if k in key_props_name:
+                    value = props_value.get(k)
+                    if k == "AgeGroup":
+                        value = value.replace("-", " to ")
+                    props_data[v] = self.map_race(value, k)
+                else:
+                    props_data[v] = props_value.get(k)
+        return key_props, props_data
+
+    def parse_file(self):
+        """
+        Converts a JSON files to data we can submit via Sheepdog. Stores the
+        records to submit in `self.summary_locations` and `self.summary_clinicals`.
+
+        Args:
+            latest_submitted_date (date): the date of latest available "summary_clinical" for project
+            url (str): URL at which the JSON file is available
+        """
+        state_summary_clinical_submitter_id = self.parse_county_data()
+        self.parse_total_state_wide(state_summary_clinical_submitter_id)
+
+    def parse_county_data(self):
+        county_vaccine_mapping = {
+            "AdministeredCount": "vaccine_administered_count",
+            "AdministeredCountChange": "vaccine_administered_count_change",
+            "AdministeredCountRollAvg": "vaccine_administered_count_roll_avg",
+            "PersonsFullyVaccinated": "vaccine_persons_fully_vaccinated",
+            "Report_Date": "date",
+            "PctVaccinatedPopulation": "vaccine_persons_fully_vaccinated_pct",
+        }
+
+        county_demo_mapping = {
+            "AgeGroup": "age_group",
+            "Race": "race",
+            "Gender": "gender",
+            "AdministeredCount": "vaccine_administered_count",
+            "PersonsFullyVaccinated": "vaccine_persons_fully_vaccinated",
+        }
+
+        inventory_reported = {
+            "LHDReportedInventory": "vaccine_LHDR_reported_inventory",
+            "CommunityReportedInventory": "vaccine_community_reported_inventory",
+            "TotalReportedInventory": "vaccine_reported_inventory",
+            "InventoryReportDate": "date",
+        }
+
+        counties = self.parse_list_of_counties()
+        state_summary_clinical_submitter_id = ""
+        for county in counties:
```

Comment:
instead of storing a separate list `counties` you could just do `for county in self.counties_inventory`